lib/src/dom/fire_event.dart

```diff
@@ -69,8 +70,10 @@ import '../user_event/user_event.dart';
 /// See: <https://testing-library.com/docs/dom-testing-library/api-events/#fireevent>
 ///
 /// {@category UserActions}
+bool fireEvent(Element element, Event event) => eventHandlerErrorCatcher(() => _fireEvent(element, event));
```

Comment:
Oh I just thought... could we add a small note in each of these doc comments saying that, unlike the JS implementation, any uncaught errors thrown during event propagation will get rethrown at the end?

I think it's important to document this behavior since it's different than in the JS